File: app/src/main/java/com/bytedance/tiktok/activity/PlayListActivity.java
package com.bytedance.tiktok.activity;

import com.bytedance.tiktok.R;
import com.bytedance.tiktok.base.BaseActivity;
import com.bytedance.tiktok.fragment.RecommendFragment;

/**
 * create by libo
 * create on 2020-05-24
 * description 视频全屏播放页
 */
public class PlayListActivity extends BaseActivity {
    public static int initPos;

    @Override
    protected int setLayoutId() {
        return R.layout.activity_play_list;
    }

    @Override
    protected void init() {
        getSupportFragmentManager().beginTransaction().add(R.id.framelayout, new RecommendFragment()).commit();
    }
}

File: app/src/main/java/com/bytedance/tiktok/activity/PlayListActivity.kt
package com.bytedance.tiktok.activity

import com.bytedance.tiktok.R
import com.bytedance.tiktok.base.BaseBindingActivity
import com.bytedance.tiktok.databinding.ActivityPlayListBinding
import com.bytedance.tiktok.fragment.RecommendFragment

/**
 * create by libo
 * create on 2020-05-24
 * description 视频全屏播放页
 */
class PlayListActivity : BaseBindingActivity<ActivityPlayListBinding>({ActivityPlayListBinding.inflate(it)}) {

    override fun init() {
        supportFragmentManager.beginTransaction().add(R.id.framelayout, RecommendFragment()).commit()
    }

    companion object {
        var initPos = 0
    }
}